src/plugin.json

```diff
@@ -37,7 +37,7 @@
     "updated": "%TODAY%"
   },
   "dependencies": {
-    "grafanaDependency": ">=8.0.0",
+    "grafanaDependency": ">=9.2.0",
```

Comment:
any reason to bump all the way to 9.2?  maybe 8.5?

Reply:
`SecretInput` is missing pre 9.1.x, so i set the dependency to 9.1.0 in [5105e8d](https://github.com/grafana/mqtt-datasource/pull/55/commits/5105e8dfbe03ba379d09803be4a83764995ab8da). i was thinking that the first official release should target a recent 9.x version since we don't have a lot of bandwidth to support 8.x issues. thoughts?

Reply:
sounds reasonable -- especially for cloud